modules/magniteAnalyticsAdapter.js

```diff
@@ -927,8 +927,8 @@ magniteAdapter.track = ({ eventType, args }) => {
       const bidStatus = args.rejectionReason === REJECTION_REASON.FLOOR_NOT_MET ? BID_REJECTED_IPF : 'rejected';
       handleBidResponse(args, bidStatus);
       break;
-    case SEAT_NON_BID:
-      handleNonBidEvent(args);
+    case PBS_ANALYTICS:
```

Comment:
Since you stop listening to non bid, is this blocked by #12044 ?

Reply:
yeah it is